### PR TITLE
Fix no goal_id bug and no header bug

### PR DESCRIPTION
**Pull request by mjyc**
Generate a GoalID or Header message when the goal_id or the header field is missing from the input goalMessage.

### CODE REVIEW

**Comment by chfritz** [comment]
:+1:
@chris-smith, @IanTheEngineer : FYI, @mjyc is a member of my team. We are starting to test rosnodejs in production, so we will probably be finding and fixing a bunch of smaller issues over the next couple of months or so. It would be great if we could get those merged. But please let me know if you would prefer us collecting them in my fork for now, so we can get those fixes out quickly, and only merge them into kinetic-devel later in one batch.

File: src/lib/ActionClient.js [hunk 1]
```diff
@@ -30,6 +31,10 @@ class ActionClient extends EventEmitter {
       GoalID = msgUtils.requireMsgPackage('actionlib_msgs').msg.GoalID;
     }
 
+    if (GoalID === null) {
```
Comment (chris-smith) [inline src/lib/ActionClient.js]: `if (Header === null) { ` ?

**Comment by chris-smith** [comment]
Awesome! We should definitely be able to merge things in - response times may vary depending on our schedule but I wouldn't expect anything to linger too long.

File: src/lib/ActionClient.js (hunk 1 above)
Comment (IanTheEngineer) [reply src/lib/ActionClient.js]: @mjyc I agree with @chris-smith here. Looks like you're intending to check and then set the `Header`. I'd be happy to merge after the change.

**Comment by IanTheEngineer** [comment]
@chfritz I'm thrilled to hear that your crew is starting to use `rosnodejs` in production, and certainly welcome any / all fixes that you uncover along the way. Both Chris and I will attempt to keep on top of issues & PR.

**Comment by chfritz** [comment]
Awesome, thanks @IanTheEngineer & @chris-smith !

File: src/lib/ActionClient.js (hunk 1 above)
Comment (mjyc) [reply src/lib/ActionClient.js]: @IanTheEngineer @chris-smith nice to meet you. I fixed the L34 as suggested.

**Comment by IanTheEngineer** [comment]
Great! Thanks for the updated commit.

**Pull request merged**